### PR TITLE
Release script follow-up work after 16.1.0-beta release

### PR DESCRIPTION
**Pull request by bvaughn**
Resolves remaining items from #11435.

Commits are atomic.

### CODE REVIEW

File: scripts/release/build-commands/print-post-build-summary.js [hunk 1]
```diff
@@ -3,16 +3,27 @@
...
   const command =
-    `./publish.js -v ${params.version}` +
-    (params.path ? ` -p ${params.path}` : '') +
-    (params.dry ? ' --dry' : '');
+    `${publishPath} -v ${version}` +
+    (path ? ` -p ${path}` : '') +
+    (dry ? ' --dry' : '');
+
+  const packagingFixturesPath = join(cwd, 'fixtures/packaging');
+  const standaloneFiturePath = join(
```
Comment (gaearon) [inline scripts/release/build-commands/print-post-build-summary.js]: Typo: fixture

File: scripts/release/build-commands/update-package-versions.js [hunk 2]
```diff
@@ -36,7 +36,13 @@ const update = async ({cwd, dry, version}) => {
...
       // These packages are auto-incremented by a minor version number.
       if (semver.lt(json.version, '1.0.0')) {
-        json.version = `0.${semver.minor(json.version) + 1}.0`;
+        const prerelease = semver.prerelease(version);
+        let suffix = '';
+        if (prerelease) {
+          suffix = `-${prerelease.join('.')}`;
+        }
+
+        json.version = `0.${semver.minor(json.version) + 1}.0${suffix}`;
```
Comment (gaearon) [inline scripts/release/build-commands/update-package-versions.js]: Not sure I understand the difference between when to use `version` and when to use `json.version`.

If I do two pre-releases in a row, does this increment minor once or twice? If it only increments once then sounds fine.

Comment (bvaughn) [reply scripts/release/build-commands/update-package-versions.js]: `version` is coming from the user's CLI param (eg `16.0.1-beta`)

The "version" in `package.json` will typically be the same, except for < 1.0 releases like the reconciler, when we'll increment the major (eg `0.1.0` -> `0.2.0`) and now also append the prerelease suffix (eg `16.1.0-beta` -> `0.2.0-beta`).

Comment (bvaughn) [reply scripts/release/build-commands/update-package-versions.js]: I'll update the inline comment to be clearer.

**Pull request merged**